addon/event_dispatcher.js

```diff
@@ -1,9 +1,19 @@
+import { assert } from '@ember/debug';
+import { getOwner } from '@ember/application';
+import { merge, assign as _assign } from '@ember/polyfills';
+import { isNone } from '@ember/utils';
+import { get, set } from '@ember/object';
 import Ember from 'ember';
 import defaultHammerEvents from './hammer-events';
 import dasherizedToCamel from './utils/string/dasherized-to-camel';
 import jQuery from 'jquery';
 import mobileDetection from './utils/is-mobile';
 
+const {
+  EventDispatcher,
+} = Ember;
```

Comment:
Ember global remains, how do we get EventDispatcher using new module syntax - its exported internally from ember-views within ember itself, but I don't find an external export?